libr/core/cmd_search.c

```diff
@@ -2253,6 +2247,54 @@ static void search_similar_pattern(RCore *core, int count) {
 	r_cons_break_pop ();
 }
 
+// Returns the intersection of two half-open intervals, [a_from, a_to) and 
+// [b_from, b_to) or [0, 0) if the intersection is empty.
+
+// valid inputs:
+// a_from <= a_to
+// b_from <= b_to
+// Issues a warning upon invalid input and does nothing.
+void range_get_intersect(ut64 a_from, ut64 a_to, ut64 b_from, ut64 b_to,
+		ut64 *out_from, ut64 *out_to) {
+	if (a_from > a_to || b_from > b_to) {
+		eprintf("WARNING: range_get_intersect called on invalid inputs!\n");
+	}
+	*out_from = R_MAX (a_from, b_from);
+	*out_to = R_MIN (a_to, b_to);
+	if (*out_from >= *out_to) {
+		*out_from = 0;
```

Comment:
So is UT64_MAX. But it doesn't matter what number it is. The range is empty. Use UT64_MAX if you want to.